controllers/score-controllers.js

```diff
@@ -0,0 +1,58 @@
+const { Hospital, Score } = require('../db/models');
+const Sequelize = require('sequelize');
+
+exports.sendHospitalScore = function(req, res) {
+  const hospitalId = req.query.hospitalId;
+  const score = req.query.score;
+  const userid = req.query.userId;
+  Score.create({
+    rate: score,
+    UserId: userid,
+    HospitalId: hospitalId,
+  }).then(() => {
+    try {
+      const Op = Sequelize.Op;
+      Score.findAll({
+        where: {
+          HospitalId: hospitalId,
+          createdAt: {
+            [Op.lt]: new Date(),
+            [Op.gt]: new Date(new Date() - 15 * 60 * 1000),
+          },
+        },
+      }).then(scores => {
+        const readableScores = scores.map(score => score.toJSON());
+        const sumScores = readableScores.reduce(
+          (acc, score) => acc + score.rate,
+          0
+        );
+        const avgScore = Math.round(sumScores / readableScores.length);
+        Hospital.findByPk(hospitalId).then(hospital => {
+          try {
+            hospital
+              .update({
+                status: avgScore,
+              })
+              .then(() => {
+                res.status(200);
+                res.json(score);
+              });
+          } catch (e) {
+            console.error(
+              `Error ${e} happened after trying to update hospital status in Hospitals`
+            );
```

Comment:
We're catching the error so the [upper catch](https://github.com/pluralengine/hospital-monitor-api/pull/6/files#diff-4b4e61620782a4acb5dc8a22faf7c17eR47) is never going to detect the error, so we won't respond to the client and we will give a timeout.
We have some options to solve this:
* Throw the error with the new information so the error will be caught by the upper catch and it will send a 500
* respond send the response here using `res.status` and `res.json`